test_data/provider_defs/user_datasets.json

```diff
@@ -1,8 +1,9 @@
 {
   "type": "DatasetLayerListingProviderDefinition",
   "id": "cbb21ee3-d15d-45c5-a175-66964adf4e85",
-  "name": "User Data Listing",
-  "description": "User specific datasets grouped by tags.",
+  "name": "Personal Data Catalog",
```

Comment:
why did you change it? Was there already some feedback?

Reply:
yes there is some feedback in Slack.